nextcord/role.py

```diff
@@ -47,22 +47,32 @@ class RoleTags:
         The bot's user ID that manages this role.
     integration_id: Optional[:class:`int`]
         The integration ID that manages the role.
+    subscription_listing_id: Optional[:class:`int`]
+        The id of this role's subscription sku and listing
+
+        .. versionadded:: 2.4
     """
 
     __slots__ = (
         "bot_id",
         "integration_id",
+        "subscription_listing_id",
         "_premium_subscriber",
+        "_available_for_purchase",
     )
 
     def __init__(self, data: RoleTagPayload):
         self.bot_id: Optional[int] = get_as_snowflake(data, "bot_id")
         self.integration_id: Optional[int] = get_as_snowflake(data, "integration_id")
+        self.subscription_listing_id: Optional[int] = get_as_snowflake(
+            data, "subscription_listing_id"
+        )
         # NOTE: The API returns "null" for this if it's valid, which corresponds to None.
         # This is different from other fields where "null" means "not there".
         # So in this case, a value of None is the same as True.
         # Which means we would need a different sentinel.
         self._premium_subscriber: Optional[Any] = data.get("premium_subscriber", MISSING)
+        self._available_for_purchase: Optional[Any] = data.get("available_for_purchase", MISSING)
```

Comment:
```suggestion
        self.available_for_purchase: bool = "available_for_purchase" in data
```

And remove the getter

Reply:
change this for _premium_subscriber too